SU-HSDO/suhumsci
# SHS-4970 mobile overlay accent bar

# READY FOR REVIEW

## Summary
Adds space to the mobile accent bar on the overlay of the carousel component. This is a different way to fix the problem from https://github.com/SU-HSDO/suhumsci/pull/1332

## Need Review By (Date)
08/23

## Urgency
medium

## Steps to Test
- Visit https://africanstudies.suhumsci.loc/community/community
- Note that the accent mark on the overlay text at mobile sizes no longer touches the text.
- Visit https://economics.suhumsci.loc/
- Note that the spacing here remains the same as on the live version

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
